client/src/components/Alerts/AlertRegister.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { useAuth, clearErrors } from '../../context/auth/AuthState';
+
+import { Toast, Row, Col } from 'react-bootstrap';
+
+const AlertRegister = ({ user, alert, setShowAlert }) => {
+    const [authState, authDispatch] = useAuth();
+    const { error } = authState;
```

Comment:
this is pretty cool to be able to do both front and backend validations once. I guess I'll read more into the dispatch and context to learn it better.  As for toast vs Native HTML5 form validation from bootstrap, I think either way is fine. It's more of eye catching vs subtle. I'm okay either way, @zldzksk1 any comments as web designer point of view?